components/python/salt/Makefile

```diff
@@ -0,0 +1,48 @@
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2018 Till Wegmueller
+#
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=salt
+COMPONENT_VERSION=2018.3.2
+COMPONENT_SUMMARY=	Portable, distributed, remote execution and configuration management system
+COMPONENT_PROJECT_URL=http://saltstack.com
+COMPONENT_FMRI=system/management/$(COMPONENT_NAME)
+COMPONENT_CLASSIFICATION=	System/Administration and Configuration
+COMPONENT_SRC=          $(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=      $(COMPONENT_SRC).tar.gz
+COMPONENT_ARCHIVE_URL=https://files.pythonhosted.org/packages/9b/84/48b46bad5fa13b47c10a71b4f58cf0f3a30fcb32b6a6599fca454b8c6256/$(COMPONENT_ARCHIVE)
```

Comment:
This looks odd to me. What's the reason for this. Salt Stacke releases are here: https://github.com/saltstack/salt/releases.

Reply:
Ah thanks i did only find the pipy URL via the documentation. even the checksum is the same. changed to the github url.